runtime/src/accounts_db.rs

```diff
@@ -1103,15 +1129,21 @@ impl AccountsDB {
         storage.0.keys().cloned().collect()
     }
 
-    pub fn process_stale_slot(&self) {
+    pub fn process_stale_slot(&self) -> usize {
+        let mut measure = Measure::start("stale_slot_shrink-ms");
+        let mut count = 0;
         if let Some(slot) = self.next_shrink_slot() {
-            self.shrink_stale_slot(slot);
+            count = self.shrink_stale_slot(slot);
         }
+        measure.stop();
+        inc_new_counter_info!("stale_slot_shrink-ms", measure.as_ms() as usize);
+
+        count
     }
 
-    pub fn shrink_all_stale_slots(&self) {
+    pub fn shrink_all_slots(&self) {
         for slot in self.all_slots_in_storage() {
-            self.shrink_stale_slot(slot);
+            self.shrink_slot_forced(slot);
```

Comment:
Hmm, I may land this later. I'm literally sick of bad snapshots.... Let's minimize the risk. ;)

Reply:
Nope, I've changed my mind. Let's just do this and rip it on the 1.2 release line (devnet/tds stage 5).